docs/metrics/learning-prometheus-server/README.md

```diff
@@ -0,0 +1,184 @@
+# Quick start on exporting metrics to Prometheus/Grafana
+
+- [Quick start on exporting metrics to Prometheus/Grafana](#quick-start-on-exporting-metrics-to-prometheusgrafana)
+  - [Prerequisite](#prerequisite)
+  - [Introduction](#introduction)
+  - [Get Prometheus](#get-prometheus)
+  - [Prometheus server](#prometheus-server)
+    - [Configuration](#configuration)
+    - [Start Prometheus](#start-prometheus)
+    - [Configure OpenTelemetry to Expose metrics to Prometheus Endpoint](#configure-opentelemetry-to-expose-metrics-to-prometheus-endpoint)
+    - [Check Results in Prometheus](#check-results-in-prometheus)
+    - [View/Query Results with Grafana](#viewquery-results-with-grafana)
+
+## Prerequisite
+
+It is highly recommended to go over the [getting-started](../getting-started/README.md)
+project under the metrics document folder before following along this document.
+
+## Introduction
+
+- [What is Prometheus?](https://prometheus.io/docs/introduction/overview/)
+
+- [Grafana support for
+  Prometheus](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph)
+
+## Get Prometheus
+
+Follow the [first steps]((https://prometheus.io/docs/introduction/first_steps/))
```

Comment:
feels vague.. We can just start with Download the latest verion of Prometheus..

Reply:
Updated.